.travis.yml

```diff
@@ -49,8 +49,8 @@ matrix:
       - cargo check --all --verbose
 
 script:
-  - cargo test --all --verbose
-  - cargo test --all --verbose --no-default-features --features std
+  - RUSTFLAGS="-C target-cpu=native" cargo test --all --verbose
```

Comment:
Do we still need this? (It does a bit a variability to the build if the CPU of the CI runner changes, so I think it would be nice to avoid it if we can.)

Reply:
How would you test aes?

Reply:
Under the new construction the check is at runtime so the AES tests will run on any hardware that supports it regardless of how the crate is compiled.